### PR TITLE
Adding Simple Tile Entity Syncing

### PR DESCRIPTION
**Pull request by nekosune**
Added instructions on simple tile entity syncing from server to client

### CODE REVIEW

**Comment by tterrag1098** [comment]
You need to add the page to the config in the main directory

**Comment by nekosune** [comment]
Changes the commit to do so

**Comment by tterrag1098** [comment]
Hm...to be consistent (and make your life easier) you should use "```" to designate code, not 4 space indents (which are a pain).

**Comment by tterrag1098** [comment]
Also this allows you to do ```java to explicitly say what syntax highlighting to use.

File: networking/SimpleTileEntities.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Simple Tile Entity Server to Client Networking
+=============================
+
+If you want to keep the client side tile entity synced with the server, you do not need the full networking system
+
+You can split the nbt saving/loading for the syncable items into their own functions
+
+    @Override
+    public void writeToNBT(NBTTagCompound tag)
```
Comment (LexManos) [inline networking/SimpleTileEntities.md]: Too much copy pasta code.

File: networking/SimpleTileEntities.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    {
+        super.writeToNBT(tag);
+        writeCustomNBT(tag);
+    }
+    
+    public void writeCustomNBT(tag)
```
Comment (LexManos) [inline networking/SimpleTileEntities.md]: Why the redirection? No explanation and just complicates things you could just call read/write directly.

File: networking/SimpleTileEntities.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        active=tag.setString("active");
+    }
+    
+Then override the getDescriptionPacket and onDataPacket
+
+    @Override
+    public Packet getDescriptionPacket()
+    {
+        NBTTagCompound nbttagcompound = new NBTTagCompound();
+        writeCustomNBT(nbttagcompound);
```
Comment (LexManos) [inline networking/SimpleTileEntities.md]: Why NBT? Why the exact same values from read/write NBT?
No explanation of what actually should be synced and why.
You're just giving copy pasta code which is BAD.

File: networking/SimpleTileEntities.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        super.onDataPacket(net,pkt);
+        readCustomNBT(pkt.getNbtCompound);
+    }
+  
+  
+Finally you need to override markDirty
+
+    @Override
+    public void markDirty()
+    {
```
Comment (LexManos) [inline networking/SimpleTileEntities.md]: Why do you need to do this? This has nothing to do with networking it's called in a couple of placed, there is no explination of what this actually is.

**Comment by LexManos** [comment]
'Put this code in your code' is not documention.

**Comment by tterrag1098** [comment]
I have to say, I agree. Documentation explains *why and how*, this is a tutorial, which is not what we are going for.

**Comment by diesieben07** [comment]
I would not call this a tutorial either.

**Pull request closed without merging**